app/presenters/brexit_taxons_presenter.rb

```diff
@@ -15,6 +15,6 @@ def call
   def featured_taxons
     @featured_taxons ||= FEATURED_TAXONS
       .map { |base_path| Taxon.find(base_path) }
-      .map { |taxon| BrexitTaxonPresenter.new(taxon) }
+      .each.with_index(1).map { |taxon, index| BrexitTaxonPresenter.new(taxon, index) }
```

Comment:
Did you mean to offset the index by 1? If not, you could use `.each_with_index`. https://stackoverflow.com/a/20258160